cmd/cored/main.go

```diff
@@ -138,6 +141,17 @@ func main() {
 	if err != nil {
 		chainlog.Fatalkv(ctx, chainlog.KeyError, err)
 	}
+	var (
+		x509Cert   *x509.Certificate
+		internalDN *pkix.Name
+	)
+	if tlsConfig != nil {
+		x509Cert, err = x509.ParseCertificate(tlsConfig.Certificates[0].Certificate[0])
```

Comment:
Nit: I think this can use `:=` and then we don't need the var declaration for this variable above.